projects/observability/src/pages/explorer/explorer.component.test.ts

```diff
@@ -164,19 +164,17 @@ describe('Explorer component', () => {
         limit: 1000,
         interval: new TimeDuration(15, TimeUnit.Second)
       }),
-      expect.objectContaining({})
+      undefined
```

Comment:
Yeah, i am not sure if the tests were working as intended before. I will double check this page to see if there is any issue.

Reply:
this seems like the only outstanding issue. If they were returning an object before and now returning undefined, something's been functionally changed. It may be on the test side, may be on the component side but seems like we should understand this issue before merging.

Reply:
Just tested this and things seem to work as expected. Not seeing any issues at all. Imo, we can go ahead with this.

Reply:
Please follow up on it - something's clearly broken, sounds like it's on the test side.